docs/topics_5/proc_web-downloading-logs-cli.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-downloading-logs-cli_{context}"]
+= Downloading pod logs using the CLI
+[role="_abstract"]
+
+
+Each pod is configured to provide detailed logging that assists with narrowing down the precise cause of an issue. You can use the CLI to download pod logs.
+
+.Procedure
+
+. Obtain the pod names:
++
+----
+$ oc get pods
```

Comment:
Just a comment that it would make more sense to specify the namespace. If someone is running lots of pods, that command will list every running pod. Not a big deal, just an observation.

Reply:
Sounds helpful. Would that be documented as:  $ oc get pods \<namespace>  ?

Reply:
You should learn how to google these things. :-) 

It wouyld be $ oc get pods -n <namespace>